client/implementation/src/main/java/io/smallrye/graphql/client/typesafe/impl/reflection/FieldInfo.java

```diff
@@ -28,6 +40,9 @@ public TypeInfo getType() {
     }
 
     public String getName() {
+        if (field.isAnnotationPresent(JsonbProperty.class) &&
+                !field.getAnnotation(JsonbProperty.class).value().isEmpty())
+            return field.getAnnotation(JsonbProperty.class).value();
```

Comment:
Shoudn't the `@Name` annotation beat the `@JsonProperty` annotation?

Reply:
Yes

Reply:
Seems you're right. I reported https://github.com/smallrye/smallrye-graphql/issues/1067